content/docs/services/s3.md

```diff
@@ -127,15 +127,16 @@ Note that "website mode" URLs don't support HTTPS, and they aren't appropriate f
 Either way, if the bucket is private, attempting to access resources will result in `AccessDenied` errors unless your application generates [pre-signed URLs](http://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html) for objects that need to be shared.
 
 ### Allowing web access from external applications
-If users wish to access their S3 buckets from outside of their cloud.gov application then set a CORS policy, which loosens security restrictions.  This is *NOT* recommended, however with a sufficiently restricted list of sites and methods can be safe:
+
+If users wish to access their S3 buckets from outside of their cloud.gov application then set a [CORS policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS), which allows the bucket to be accessed by requests made by JavaScript code run in the context of other third-party web services. You can set a CORS policy allowing arbitrary origins to read bucket data, or you can restrict access to a specific list of sites and methods:
```

Comment:
@jmcarp Great point. I gave it a full rewrite and added more explanation. Mind giving it another review?